notebooks/07_emg_hand_gesture_classification.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
<p>Maybe add some explanation of the data here?</p><ul><li>time is given in microseconds?</li><li>c0-c7 represent the electrode</li><li>the label is "rest" for resting the arm</li><li>exp is the ....</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/geomstats/geomstats/pull/899/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>